transports/wasm-ext/src/webtransport.rs

```diff
@@ -0,0 +1,11 @@
+mod cached_js_promise;
+mod connection;
+mod error;
+mod substream;
+mod transport;
+mod utils;
+
+pub use self::connection::Connection;
+pub use self::error::Error;
+pub use self::substream::Substream;
```

Comment:
```suggestion
pub use self::substream::Stream;
```

See https://github.com/libp2p/rust-libp2p/pull/3912. We are working on moving away from "Substream".